lib/publiccloud/acr.pm

```diff
@@ -0,0 +1,48 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+#
```

Comment:
i think you miss `# SPDX-License-Identifier: FSFAP`. check the link i sent you again

Reply:
Done. I also updated the license in ecr and gcr

Reply:
Thanks